gradle.properties

```diff
@@ -1 +1,2 @@
 org.gradle.java.installations.auto-download=false
+mockbukkit.version=
```

Comment:
Would it make sense to call the default version `SNAPSHOT` in case you would publish to your own local maven repository?